packages/Symfony/tests/Rector/BinaryOp/ResponseStatusCodeRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\BinaryOp\ResponseStatusCodeRector\Fixture;
+
+class SomeController
+{
+    public function index()
+    {
+        $response = new \Symfony\Component\HttpFoundation\Response();
+        $response->setStatusCode(200);
+
+        if ($response->getStatusCode() === 200) {}
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\Rector\BinaryOp\ResponseStatusCodeRector\Fixture;
+
+class SomeController
+{
+    public function index()
+    {
+        $response = new \Symfony\Component\HttpFoundation\Response();
+        $response->setStatusCode(\Symfony\Component\HttpFoundation\Response::HTTP_OK);
+
+        if ($response->getStatusCode() === \Symfony\Component\HttpFoundation\Response::HTTP_OK) {}
+    }
+}
+
+?>
```

Comment:
Nice and simle test coverring all it needs with minimal code :+1:  

Great job! :clap: